packages/sanity/playwright-ct.config.ts

```diff
@@ -38,17 +38,8 @@ export default defineConfig({
         ],
       ],
 
-  /* Maximum time one test can run for. */
-  timeout: 60 * 1000,
```

Comment:
The default option is 30s, it's better to allow specific tests to run longer if they need to, than to have long global defaults.